src/wallet/wallet.h

```diff
@@ -986,4 +984,36 @@ class CAccount
     }
 };
 
+static CCriticalSection cs_wallets;
+class CWallets
+{
+public:
+    static std::vector<CWallet *> wallets;
```

Comment:
Yes. I had a `Namespace` at the beginning and though that there is no big need for a Class or even private static class variables. But yes, why not private. Will update.